fuel-core-interfaces/src/relayer.rs

```diff
@@ -0,0 +1,143 @@
+use std::collections::HashMap;
+
+use async_trait::async_trait;
+use fuel_storage::Storage;
+use fuel_types::{Address, AssetId, Bytes32, Word};
+use serde::{Deserialize, Serialize};
+use tokio::sync::oneshot;
+
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub struct DepositCoin {
+    pub owner: Address,
+    pub amount: Word,
+    pub asset_id: AssetId,
+    pub eth_block_deposited: u64,
```

Comment:
```suggestion
    pub base_chain_block_deposited: u64,
```

Reply:
i renamed it to `deposited_da_height` so it is in same naming as others feilds.